MANIFEST.in

```diff
@@ -1,4 +1,6 @@
 recursive-include pynxtools/definitions/base_classes/ *.xml
 recursive-include pynxtools/definitions/applications/ *.xml
 recursive-include pynxtools/definitions/contributed_definitions/ *.xml
-include pynxtools/definitions/ *.xsd
+include pynxtools/definitions/*.xsd
```

Comment:
if this line needed to be reformatted, why is not the reformatting of the above lines of *.xml necessary?

Reply:
The line was wrongly formatted and no `xsd` files were included in the site-packages. This is different for `recursive-include` (with space in between) and `include` (without space in between).